packages/datadog-plugin-amqp10/test/integration-test/client.spec.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const {
+  FakeAgent,
+  createSandbox,
+  checkSpansForServiceName,
+  skipUnsupportedNodeVersions,
```

Comment:
We'll probably want to rename this method. Specifically, it comes from a file named `integration-tests/helpers`, and the method name is `skipUnsupportedNodeVersions`, so there's no indicator that the check is for ESM related code. So an engineer that comes along and sees this method might think that we globally skip tests always if the version of node is 20.

Instead, maybe a name like `esmTestSkipper`, to better convey they purpose and scope.